### ISSUE TITLE
Can't change download directory to any path on external sd card

### ISSUE THREAD

**Issue by boscowitch** [clementine-player/Android-Remote#79]
Hi I can't download to my extSdCard folder it for example:
/storage/extSdCard/Clementine

it just says that it didnt find a "memory device"! (in german no shure about the correct unlocalised spelling)

**Comment by amuttsch** [comment]
Can anyone how encounted this problem report if this problem is still present?

**Comment by akyag** [comment]
Yes.. This problem is still present. Crashes everytime I click on download folder path. I am on android 5.1.1 and Remote version 10.1

**Comment by OkunH** [comment]
I use android 6, honor 6. When i change in the settings to the folder in the SD-card, where is a copy of my music is stored, there comes the notification: "verzeichnis nicht beschschreibbar" (can´t write to the folder).
Afaik other apps are able to write to other folders?!

**Comment by sbrl** [comment]
I've just stumbled upon this issue too. Every other app on my phone can write to arbitrary folders on my external SD Card - except the clementine remote. The special `/storage/extSdCard/Android/data/de.qspool.clementineremote/files/Music` folder appears to work though.

**Comment by bha2020** [comment]
using 6.0 onwards Im able to read my removable sdcard but unable to write to sdcard.
But how shareIt app manages to write to sdcard .Is there any solution like that .
Please help

**Comment by pestsov** [comment]
This makes the download function useless on non rooted phones, since /Android/data/ is not accessible!
I believe there must be a way to rewrite the app so download to /Download or other folder works..

**Comment by Rasie1** [comment]
Pls fix this, it's 5 minutes of work and it will make the download functionality not useless. Make the directory selectable
